examples/2d/plot_filters.py

```diff
@@ -54,29 +54,30 @@ def colorize(z):
     filter_c = np.fft.fftshift(filter_c)
     axs[i // L, i % L].imshow(colorize(filter_c))
     axs[i // L, i % L].axis('off')
-    axs[i // L, i % L].set_title("$j = {}$ \n $\\theta={}".format(i // L, i % L))
+    axs[i // L, i % L].set_title("$j = {}$ \n $\\theta={}$".format(i // L, i % L))
```

Comment:
I think globally the procedural version, `plt.subplot(J, L, i + 1)` would have let you get away with much less code.

But this object-oriented version is fine too, since it makes things explicit

Reply:
yeah no I'm fine with this. Was just saying it could be shorter, but it doesn't have to be.